.gitignore

```diff
@@ -32,3 +32,7 @@ local/config/vufind/*.ini
 local/config/vufind/*.json
 local/config/vufind/*.key
 local/config/vufind/*.yaml
+
+.idea/*
+hebis_local/*
+logs/*
```

Comment:
I think these changes may have been committed by accident.

Reply:
I fixed this change.